fe/fe-core/src/main/java/com/starrocks/analysis/KeysDesc.java

```diff
@@ -78,6 +78,9 @@ public void analyze(List<ColumnDef> cols) throws SemanticException {
         for (int i = 0; i < keysColumnNames.size(); ++i) {
             String name = cols.get(i).getName();
             if (!keysColumnNames.get(i).equalsIgnoreCase(name)) {
+                if (!cols.contains(keysColumnNames.get(i))) {
```

Comment:
cols is a List<ColumnDef>, and kyesColumNames is a String.
how does `cols.contains` work.

And column should be case insensitive, does it work?

Reply:
the keyColumnNames is a list of string，get(i) will return a string，and I had test this code by building and testing the FE. I forgot the case insentivity，I will fix it later，thank you!

Reply:
> cols is a List, and kyesColumNames is a String. how does `cols.contains` work.
> 
> And column should be case insensitive, does it work?

sorry, I read the code again, You are right, cols.contains not work, i made a mistake, and i had fixed it, thank you!